aries_cloudagent/core/conductor.py

```diff
@@ -70,6 +70,7 @@
 from .util import STARTUP_EVENT_TOPIC, SHUTDOWN_EVENT_TOPIC
 
 LOGGER = logging.getLogger(__name__)
+OUTBOUND_STATUS_PREFIX = "acapy::outbound-message::"
```

Comment:
We talked about this one previously. I think I know where I'd like to see this now. I the same module that contains the `OutboundSendStatus` enum, let's put this constant and then let's add a method to the class that returns the topic from the status enum. Something along the lines of:

```python
OUTBOUND_STATUS_TOPIC_PREFIX = ...

class OutboundSendStatus(Enum):
    # enum defs here....
    
    @property
    def topic(self):
        return f"{OUTBOUND_STATUS_TOPIC_PREFIX}{self.value}"
```